home/src/main/resources/rdf/accessControl/firsttime/template_update_related_allowed_property.n3

```diff
@@ -228,11 +229,24 @@
     access:requiresCheck :RelationCheck ;
     .
 
+:AllowxternalIdMatchingPropertyStatement a access:Rule;
```

Comment:
This is the 'External' typo @wwelling  was referring to, though it should be functional anyway as long as its consistent.

Reply:
Thanks! Fixed in https://github.com/vivo-project/Vitro/pull/450/commits/114761eb58e25e381693d2865b4595ee4ed1a927